SharePointFramework/ProjectWebParts/src/components/ProjectNews/ProjectNews.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  Button,
+  FluentProvider,
+  IdPrefixProvider,
+  Spinner
+} from '@fluentui/react-components'
+import { NewsRegular, NewsFilled, bundleIcon } from '@fluentui/react-icons'
+import React, { FC } from 'react'
+import { format } from '@fluentui/react'
+
+import { IProjectNewsProps } from './types'
+import { customLightTheme, UserMessage, WebPartTitle } from 'pp365-shared-library'
+import { useProjectNews } from './useProjectNews'
+import { ProjectNewsContext } from './context'
+import ProjectNewsDialog from './ProjectNewsDialog/NewsDialog'
+import strings from 'ProjectWebPartsStrings'
+import styles from './ProjectNews.module.scss'
+import RecentNewsList from './ProjectNewsRecentNewsList/RecentNewsList'
+import { useProjectNewsDialog } from './ProjectNewsDialog/useProjectNewsDialog'
+
+
+export const ProjectNews: FC<IProjectNewsProps> = (props) => {
+  const { context, fluentProviderId } = useProjectNews(props)
+  const dialog = useProjectNewsDialog(props)
+  const recentNews = context.state.data?.news || []
+  const { loading, error } = context.state
+  const NewsIcon = bundleIcon(NewsFilled, NewsRegular)
```

Comment:
Legg til ikon for dette i `iconCatalog` i `shared` og anvend med `getFluentIcon`...